src/uk/gov/hmcts/contino/AbstractBuilder.groovy

```diff
@@ -36,8 +36,8 @@ abstract class AbstractBuilder implements Builder, Serializable {
   def setupToolVersion() {
 
     //Setting Up JAVA_HOME here as its used for Sonar scan even in YarnBuilder.
-    if (steps.fileExists("/usr/share/jdk-11.0.2")) {
-      steps.env.JAVA_HOME = "/usr/share/jdk-11.0.2"
+    if (steps.fileExists("/usr/lib/jvm/java-11-openjdk")) {
```

Comment:
Is this even needed when there’s only 1 installed?

Reply:
There is still java 8 on the VM, couldn’t trace it down.

Reply:
Merging it as would like to test couple of builds before others start using Jenkins.